src/components/BalanceBlock/BalanceBlock.component.tsx

```diff
@@ -70,7 +71,7 @@ const BalanceBlockContent = ({
   }
 
   return (
-    <Typography variant="h5" sx={{ mt: 1 }}>
+    <Typography variant={variant} sx={{ mt: 1 }}>
```

Comment:
shouldn't we pass in the props component which should we display? as headers in those places aren't semantic :D It's worth thinking about it in the future

Reply:
You're right, we could create a ticket to make the page more semantic and work on it when all features are ready. I think we will add all needed props while working on it